scripts/install/qt_mac_installer.sh

```diff
@@ -93,6 +93,8 @@ install_name_tool -change @rpath/QtXml.framework/Versions/A/QtXml @rpath/Content
 install_name_tool -rpath @loader_path/../lib @loader_path/../.. lupdate
 install_name_tool -change @rpath/QtCore.framework/Versions/A/QtCore @rpath/Contents/Frameworks/QtCore.framework/Versions/A/QtCore lupdate
 install_name_tool -change @rpath/QtXml.framework/Versions/A/QtXml @rpath/Contents/Frameworks/QtXml.framework/Versions/A/QtXml lupdate
+install_name_tool -change @rpath/QtQml.framework/Versions/A/QtQml @rpath/Contents/Frameworks/QtQml.framework/Versions/A/QtQml lupdate
+install_name_tool -change @rpath/QtNetwork.framework/Versions/A/QtNetwork @rpath/Contents/Frameworks/QtNetwork.framework/Versions/A/QtNetwork lupdate
```

Comment:
ditto about updating https://github.com/cyberbotics/webots/wiki/Qt-compilation for explanations.